proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -280,6 +280,11 @@ message WorkflowExecutionInfo {
     // Should be deprecated once the transition history is fully launched
     repeated VersionedTransition previous_transition_history = 100;
     VersionedTransition last_transition_history_break_point = 101;
+
+    // This is a set of child workflows that were initialized after the reset point in the parent workflow.
+    // The children are identified by the key "workflow_type:workflow_id". When the parent starts to make progress after reset, it uses this data to
+    // determine the right start policy to apply to the child.
+    map<string, bool> children_initialized_post_reset_point = 102;
```

Comment:
Just calling out that this new field won't be replicated with today's replication stack. So the new feature will only work properly with in single dc setup

Reply:
Yes. I will be adding this and a few other fields (`reset_run`) to replication stack.